app/MenuTitle.java

```diff
@@ -47,4 +47,8 @@ public static final void displayOptions(ArrayList<String> options) {
         }
         System.out.println("");
     }
+
+    public static final String getErrorMessage(String errorMessage) {
+        return (Colors.RED + errorMessage + Colors.RESET);
+    }
```

Comment:
what if instead of getting the errorMessage, it just prints it out nalang na appended na ang `Colors.RED` and `Colors.RESET`?

although ma revert almost everything u did here, i think mas cleaner, kay ang mga strings kay ma preserve to be just strings and sa mga catch blocks nalang mag

```
MenuTitle.printErrorMessage(e.getMessage()); // or smth like that
```

watchu think

Reply:
oki thank you